src/Compilers/Core/MSBuildTaskTests/GenerateMSBuildEditorConfigTests.cs

```diff
@@ -97,6 +97,34 @@ public void MultipleItemMetaDataCreatesSections()
 ", result);
         }
 
+        [Fact]
```

Comment:
Can you add a WorkItem attribute for this?

Reply:
Done.